src/iuse.cpp

```diff
@@ -5227,6 +5227,20 @@ cata::optional<int> iuse::hotplate( Character *p, item *it, bool, const tripoint
     return cata::nullopt;
 }
 
+cata::optional<int> iuse::hotplate_atomic( Character *p, item *it, bool, const tripoint & )
+{
+    if( p->is_mounted() ) {
+        p->add_msg_if_player( m_info, _( "You can't do that while mounted." ) );
+        return cata::nullopt;
+    }
+    
```

Comment:
```suggestion

```

I didn't notice before, this line has trailing whitespace. Removing the spaces on this line should allow the style check to pass.